pytest_ordering/__init__.py

```diff
@@ -35,72 +36,44 @@ def pytest_configure(config):
 
 
 def pytest_collection_modifyitems(session, config, items):
-    items[:] = list(_order_tests(items))
+    grouped_items = {}
 
+    for item in items:
 
-def orderable(marker_name, marker_info):
-    if not hasattr(marker_info, 'kwargs'):
-        return False
-    elif 'order' in marker_info.kwargs:
-        return True
-    else:
-        match = re.match('^order(\d+)$', marker_name)
-        return bool(match) or marker_name in replacements
+        for mark_name, order in orders_map.items():
```

Comment:
It's not possible to raise error for two or more ```@pytest.mark.run(order=<val>)``` marks, because pytest overwrite all previous marks with the same names, and it's normal.

```python
import pytest


@pytest.mark.run(order=2)
@pytest.mark.run(order=1)
def test_1(): pass

```

```
ipdb> item.keywords.items()
[('ordering', True), ('proba.py', True), ('run', <MarkInfo 'run' args=() kwargs={'order': 2}>), ('test_1', True)]
```